LFGBulletinBoard/GroupBulletinBoard.lua

```diff
@@ -124,12 +124,16 @@ function GBB.LevelRange(dungeon,short)
 end
 
 function GBB.FilterDungeon(dungeon, isHeroic, isRaid)
-	if dungeon==nil then return false end
+	if dungeon == nil then return false end
 	if isHeroic == nil then isHeroic = false end
 	if isRaid == nil then isRaid = false end
+
+	-- If the user is within the level range, or if they're max level and it's heroic.
+	local inLevelRange = (not isHeroic and GBB.dungeonLevel[dungeon][1] <= GBB.UserLevel and GBB.UserLevel <= GBB.dungeonLevel[dungeon][2]) or (isHeroic and GBB.UserLevel == 70)
 	
-	return GBB.DBChar["FilterDungeon"..dungeon] and (isRaid or ((GBB.DBChar["HeroicOnly"] == false or isHeroic) and (GBB.DBChar["NormalOnly"] == false or isHeroic == false))) and
-		(GBB.DBChar.FilterLevel==false or (GBB.dungeonLevel[dungeon][1] <= GBB.UserLevel and GBB.UserLevel <= GBB.dungeonLevel[dungeon][2]))
+	return GBB.DBChar["FilterDungeon"..dungeon] and 
+		(isRaid or ((GBB.DBChar["HeroicOnly"] == false or isHeroic) and (GBB.DBChar["NormalOnly"] == false or isHeroic == false))) and
```

Comment:
Causes raids to be tagged as heroic some of the time. Will take more of a look to see the proper fix

Reply:
Actually this is not your fault